client/src/component/common/MainButton/index.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import './style.css';
+import { Button } from 'antd';
+import PropTypes from 'prop-types';
+
+const MainButton = ({
+  backgroundColor,
+  border,
+  color,
+  padding,
+  Buttontext,
```

Comment:
Change this to `children` so we could get the benefit of the `children` attribute like this
```jsx
<MainButton>بحث</MainButton>
```